### PR TITLE
Address some minor pending TODO items

### PR DESCRIPTION
**Pull request by rspurgeon**

### CODE REVIEW

File: docs/event-processing/dead-letter-stream.md [hunk 1]
```diff
@@ -6,7 +6,6 @@ seo:
 
 # Dead Letter Stream
 
-<!-- TODO: Event Processing Application is a pattern that should be linked to in the following text -->
 [Event Processing Applications](event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
```
Comment (ybyzek) [inline docs/event-processing/dead-letter-stream.md:9]:
```suggestion
[Event Processing Applications](../event-processing/event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
```

**Comment by ybyzek** [comment]
@rspurgeon  Either here on in another PR, can we fix these references to be consistent with the other syntax, by not assuming same directory

`./event-deserializer.md` --> `../event/event-deserializer.md`


```
event/event-serializer.md:machine-readable reference point for [Event Deserializers](./event-deserializer.md) 
event/event-serializer.md:* An [Event Standardizer](./event-standardizer.md) can reformat
event/event-serializer.md:  with a [Schema-on-Read](./schema-on-read.md) strategy.
event/event-serializer.md:* The counterpart of an event serializer (for writing) is an [Event Deserializer](./event-deserializer.md) (for reading).
event/event-serializer.md:  [Data Contracts](./data-contract.md), in which we want to adhere to a
event/event-deserializer.md:Serializer](./event-serializer.md). Still, the story is certainly not
event/event-deserializer.md:For example, in the [Event Serializer](./event-serializer.md) pattern
event/event-deserializer.md:Schema. See [Event Serializer](./event-serializer.md) for a discussion
event/event-deserializer.md:* An [Event Standardizer](./event-standardizer.md) can reformat
event/event-deserializer.md:  in code with a [Schema-on-Read](./schema-on-read.md) strategy.
event/event-deserializer.md:* The counterpart of an event deserializer (for reading) is an [Event Serializer](./event-serializer.md) (for writing).
event/event-deserializer.md:  Contracts](./data-contract.md), in which we want to adhere to a
event-processing/content-filter.md:Application](event-processing-application.md) can often be very
event-processing/content-filter.md:Create an [Event Processor](./event-processor.md) that inspects each
event-processing/dead-letter-stream.md:[Event Processing Applications](event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
event-source/database-write-aside.md:* See [Database Write Through](database-write-through.md) for an alternative example of writing database changes to an Event Stream
event-source/database-write-through.md:* See [Database Write Aside](database-write-aside.md) for an alternative example of writing database changes to an Event Stream
event-storage/limited-retention-event-stream.md:* A related pattern is the [Infinite Retention Event Stream](infinite-retention-event-stream.md) pattern which details Event Streams that stores events indefinitely.
event-storage/compacted-event-stream.md:It is important to understand that compaction, on purpose, removes historical data from an event stream by removing superseded events as defined above. In many use cases, however, historical data should not be removed, such as for a stream of financial transactions, where every single transaction needs to be recorded and stored. Here, if the storage of the stream is the primary concern, use an [Infinite Retention Event Stream](infinite-retention-event-stream.md) instead of a compacted stream.
event-stream/schema-evolution.md:How can I restructure or add new information to an event, ideally in a way that ensures [Schema Compatibility](schema-compatibility.md)?
```

File: docs/event-processing/dead-letter-stream.md (hunk 1 above)
Comment (rspurgeon) [reply docs/event-processing/dead-letter-stream.md:9]: It's in the same folder, the link works as is, i think the extra characters aren't necessary

**Comment by rspurgeon** [comment]
> @rspurgeon (related to comment above) either here on in another PR, can we fix these references to be consistent with the other syntax, by not assuming same directory
> 
> `./event-deserializer.md` --> `../event/event-deserializer.md`
> 
> ```
> event/event-serializer.md:machine-readable reference point for [Event Deserializers](./event-deserializer.md) 
> event/event-serializer.md:* An [Event Standardizer](./event-standardizer.md) can reformat
> event/event-serializer.md:  with a [Schema-on-Read](./schema-on-read.md) strategy.
> event/event-serializer.md:* The counterpart of an event serializer (for writing) is an [Event Deserializer](./event-deserializer.md) (for reading).
> event/event-serializer.md:  [Data Contracts](./data-contract.md), in which we want to adhere to a
> event/event-deserializer.md:Serializer](./event-serializer.md). Still, the story is certainly not
> event/event-deserializer.md:For example, in the [Event Serializer](./event-serializer.md) pattern
> event/event-deserializer.md:Schema. See [Event Serializer](./event-serializer.md) for a discussion
> event/event-deserializer.md:* An [Event Standardizer](./event-standardizer.md) can reformat
> event/event-deserializer.md:  in code with a [Schema-on-Read](./schema-on-read.md) strategy.
> event/event-deserializer.md:* The counterpart of an event deserializer (for reading) is an [Event Serializer](./event-serializer.md) (for writing).
> event/event-deserializer.md:  Contracts](./data-contract.md), in which we want to adhere to a
> event-processing/content-filter.md:Application](event-processing-application.md) can often be very
> event-processing/content-filter.md:Create an [Event Processor](./event-processor.md) that inspects each
> event-processing/dead-letter-stream.md:[Event Processing Applications](event-processing-application.md) may encounter invalid data as they operate over the infinite stream of events. Errors may include invalid data formats, nonsensical, missing or corrupt values, technical failures, or other unexpected scenarios.
> event-source/database-write-aside.md:* See [Database Write Through](database-write-through.md) for an alternative example of writing database changes to an Event Stream
> event-source/database-write-through.md:* See [Database Write Aside](database-write-aside.md) for an alternative example of writing database changes to an Event Stream
> event-storage/limited-retention-event-stream.md:* A related pattern is the [Infinite Retention Event Stream](infinite-retention-event-stream.md) pattern which details Event Streams that stores events indefinitely.
> event-storage/compacted-event-stream.md:It is important to understand that compaction, on purpose, removes historical data from an event stream by removing superseded events as defined above. In many use cases, however, historical data should not be removed, such as for a stream of financial transactions, where every single transaction needs to be recorded and stored. Here, if the storage of the stream is the primary concern, use an [Infinite Retention Event Stream](infinite-retention-event-stream.md) instead of a compacted stream.
> event-stream/schema-evolution.md:How can I restructure or add new information to an event, ideally in a way that ensures [Schema Compatibility](schema-compatibility.md)?
> ```

I can change the ones to have consistent 'same directory' paths (without the `.`), but I don't think we need the up a directory and back into the same directory unecessarily

**Comment by ybyzek** [comment]
>  I don't think we need the up a directory and back into the same directory unecessarily

It's not required but helps with consistency.

**Comment by rspurgeon** [comment]
> > I don't think we need the up a directory and back into the same directory unecessarily
> 
> It's not required but helps with consistency.

IMO this isn't necessary consistency, it's just extra uneeded characters in the files

**Comment by rspurgeon** [comment]
Making an executive decision to merge this w/ out green check

**Pull request merged**